ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py

```diff
@@ -729,3 +729,40 @@ def test_jax_numpy_std(
         atol=1e-3,
         rtol=1e-3,
     )
+
+
+# median
+@handle_frontend_test(
+    fn_tree="jax.numpy.median",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
the function jax.numpy.median supports more types besides just float. We should try to be as comprehensive as possible. Please try "valid" here to include all valid dtypes for the tests.

Reply:
Thank @Ishticode . When I change to "valid" I get an error "the ground truth framework jax returned a float32 datatype while the backend numpy returned a float64 datatype". But the data input is uint8. I have stuck with this error. Do you have any recomemd for me? What should I try? 
![Screenshot from 2023-03-27 17-09-11](https://user-images.githubusercontent.com/44942645/227912959-96f65b98-3707-498b-b75f-8b0f5e8f814d.png)